DDDEastAnglia.DatabaseMigrations/Migrations/3-AddSponsorsTable.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentMigrator;
+
+namespace DDDEastAnglia.DatabaseMigrations.Migrations
+{
+    [Migration(20150411)]
+    public class AddSponsorsTable : Migration
+    {
+        public override void Up()
+        {
+            Create.Table("Sponsors")
+                .WithColumn("SponsorId").AsInt32().NotNullable().Identity().PrimaryKey()
+                .WithColumn("Name").AsString(50).NotNullable()
+                .WithColumn("Url").AsString(255).NotNullable()
+                .WithColumn("SponsorshipAmount").AsInt32().Nullable()
+                .WithColumn("Logo").AsBinary(int.MaxValue).NotNullable()
+                .WithColumn("PaymentDate").AsDateTime().Nullable()
+                .WithColumn("ShowPublicly").AsBoolean().NotNullable();
+        }
+
+        public override void Down()
+        {
+            Delete.Table("Sponsors");
```

Comment:
I think so, as if we were to roll back there is no point of having the sponsor details in the database as they are of no use.